### PR TITLE
Emilien's stuff (mostly for pixel tracks)

### PR DESCRIPTION
**Pull request by echapon**
In this PR:
- Most important: add pixel tracks to ggHiNtuplizer
- Some simple tools in analysis that I used for development
- many .pyc files which are just noise (should probably just be removed from git)

### CODE REVIEW

**Comment by echapon** [comment]
@jniedzie please have a look. I'll make another PR soon with the stat / limit setting code.

**Comment by jniedzie** [comment]
Hi @echapon,

could you please add *.pyc to gitignore and remove those files from the PR? There are 235 files in the PR, which makes it really difficult to review ;)

File: analysis/prepareBasicPlots.cpp [hunk 1]
```diff
@@ -475,6 +475,9 @@ void fillLbLHistograms(Event &event, const map<string, TH1D*> &hists, EDataset d
   fillNoiseHists(   event, hists, name, "lbl", suffix);
   fillNoiseHists(   event, hists, name, "lbl", "all");
   fillTracksHists(  event, hists, dataset, "pass_lbl");
+
+  // we found a good event!! cool, dump it to the log
+  Log(0) << "PASSING! Run=" << event.GetRunNumber() << " LS=" << event.GetLumiSection() << " Evt=" << event.GetEventNumber() << " aco=" << aco << "\n";
```
Comment (jniedzie) [inline analysis/prepareBasicPlots.cpp:480]: Maybe make it `Log(1)` or `Log(2)`, as this is for debugging only I guess.

**Comment by echapon** [comment]
Changes as suggested. I did rebase as you can see in the git history, but somehow it failed on ggHiNtuplizer and some bits were lost...

**Pull request merged**